MicrosoftFluentUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'MicrosoftFluentUI'
-  s.version          = '0.12.0'
+  s.version          = '0.13.0'
```

Comment:
Realized we need to update the version in one more place: `MicrosoftFluentUI.podspec`.

Reply:
where?